hdx_hapi/db/models/views/all_views.py

```diff
@@ -176,21 +210,20 @@ class HumanitarianNeedsView(Base):
     admin2_is_unspecified: Mapped[bool] = column_property(humanitarian_needs_view.c.admin2_is_unspecified)
     admin1_ref: Mapped[int] = column_property(humanitarian_needs_view.c.admin1_ref)
 
-
 class LocationView(Base):
     __table__ = location_view
     id: Mapped[int] = column_property(location_view.c.id)
     code: Mapped[str] = column_property(location_view.c.code)
     name: Mapped[str] = column_property(location_view.c.name)
+    from_cods: Mapped[bool] = column_property(location_view.c.from_cods)
     reference_period_start: Mapped[DateTime] = column_property(location_view.c.reference_period_start)
     reference_period_end: Mapped[DateTime] = column_property(location_view.c.reference_period_end)
 
-
 class NationalRiskView(Base):
     __table__ = national_risk_view
     resource_hdx_id: Mapped[str] = column_property(national_risk_view.c.resource_hdx_id)
     location_ref: Mapped[int] = column_property(national_risk_view.c.location_ref)
-    risk_class: Mapped[RiskClass] = column_property(national_risk_view.c.risk_class)
+    risk_class: Mapped[str] = column_property(national_risk_view.c.risk_class)
```

Comment:
The type here should be `RiskClass`. The `all_view.py` from the `dev` branch should be the correct one.